src/model/Models.ts

```diff
@@ -40,6 +40,35 @@ export interface ITeleop {
 	Climb: string;
 }
 
+export interface IMatch {
```

Comment:
We want this object to match the shape that the server is expecting. We can copy this from last year:
```
export interface IMatch {
	gameYear: number;
	eventCode: string;
	matchNumber: string;
	robotNumber: string;
	creator: string;
	allianceColor: string;
	objectives: IObjective[];
}

export interface IObjective {
	gamemode: Gamemode,
	objective: string;
	count: number;
	list?: number[];
}
```